### PR TITLE
Clean up `Storyboard/Scripting/General rules`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Things I did:

- grammar issues fixed
- styling improved
- removed `you`s

Besides this, I also converted some bullet point lists into paragraphs were plausible, as I think it improves readability.

- [x] @ maintainers (or someone else knowledgeable): Check if the mention of `Taiko` and `Catch the Beat` is intentional (--> `osu!taiko` and `osu!catch` instead).
- [x] Keyboard shortcuts are placed into two `` ` ``. I'm not sure if this applies to image descriptions as well (see `SBS_Time.jpg`).

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Storyboard/Scripting/General_Rules/en.md [hunk 1]
```diff
@@ -2,103 +2,109 @@
...
+Animations are done in-engine, so the PNG layer system or animation features should not be used. Instead, save each frame as a separate file and name the files with a decimal number before the extension (e.g., "sample0.png", "sample1.png" for a 2-frame animation "sample.png").
 
 ### Screen size
 
-![Editor screen size. Green is screen size and Red is playarea](img/SBS_SS.jpg "Editor screen size. Green is screen size and Red is playarea")
+![Editor screen size. Green is screen size and Red is play area](img/SBS_SS.jpg "Editor screen size. Green is screen size and Red is play area")
+
+The editor screen is 640 x 480 pixels and the general play area is 510 x 385 pixels.
+
+Coordinates are specified with positive values for `X` going to the **right**, positive values for `Y` going **down**, and the origin (0,0) being placed at the upper-left corner of the screen. This is different from traditional [Euclidean coordinate systems](https://en.wikipedia.org/wiki/Euclidean_space), but is the same as most computer-graphics systems. It is possible to specify coordinates to be outside of these boundaries (e.g., for a sprite to come in from off-screen).
```
Comment (The-Last-Cookie) [inline wiki/Storyboard/Scripting/General_Rules/en.md:25]: Not sure if it's worth to mention euclidean space here at all.

File: wiki/Storyboard/Scripting/General_Rules/en.md [hunk 2]
```diff
@@ -2,103 +2,109 @@
...
 
-- The screen is 640 pixels wide by 480 pixels tall `(640x480)`.
-  - General playarea is 510 pixels wide by 385 pixels tall `(510x385)`.
-- Coordinates are specified with X going to the **right**, Y going **down**, and the origin (0,0) at the upper-left corner of the screen. This is different from traditional Euclidean coordinate systems, but is the same as most computer-graphics systems.
-- You are allowed to specify coordinates outside of these boundaries (e.g., for a sprite to come in from off-screen).
-- The coordinates of your cursor will be displayed in the Design tab in the [Beatmap Editor](/wiki/Client/Beatmap_editor).
+The cursor's coordinates will be displayed in the [Design tab](/wiki/Client/Beatmap_editor/Design) of the [Beatmap Editor](/wiki/Client/Beatmap_editor).
```
Comment (The-Last-Cookie) [inline wiki/Storyboard/Scripting/General_Rules/en.md:27]: Not sure about this one here either. The coordinates are visible in both the `compose` and the `design` tab, as visible in the picture above.

File: wiki/Storyboard/Scripting/General_Rules/en.md [hunk 3]
```diff
@@ -2,103 +2,109 @@
...
 
 **Editor coordinates:**
```
Comment (The-Last-Cookie) [inline wiki/Storyboard/Scripting/General_Rules/en.md:29]: Level 4 heading or emphasis? 🤔

File: wiki/Storyboard/Scripting/General_Rules/en.md [hunk 4]
```diff
@@ -2,103 +2,109 @@
...
+All storyboard sprites are placed below the skin and [hit objects](/wiki/Gameplay/Hit_object). So, even the "highest" (Foreground) layer in the storyboard will always be behind the HP bar, the circles/sliders/spinners, the cursor, etc.
+
+These are the four storyboard layers, in increasing order of priority:
+
+- Background
+- Fail (only displayed if the player is in the "Fail state", see [Game State](#game-state) below)
+- Pass (only displayed if the player is in the "Pass state", see [Game State](#game-state) below)
+- Foreground
+
+Note that the "Fail" and "Pass" layers are never on-screen simultaneously, unlike in the design tab.
```
Comment (The-Last-Cookie) [inline wiki/Storyboard/Scripting/General_Rules/en.md:47]:
```suggestion
Note that the "Fail" and "Pass" layers are never on-screen simultaneously, unlike in the `Design` tab.
```

File: wiki/Storyboard/Scripting/General_Rules/en.md [hunk 5]
```diff
@@ -2,103 +2,109 @@
...
   - This includes events that are on **BOTH** the Pass/Fail layers, even though only one will be displayed.
-    - Example: If your Fail storyboard ends at time 20000 and your Pass storyboard ends at time 25000, the game will wait until time 25000 even if the player is in the Fail State (all objects will disappear). Therefore, it's best to ensure that both Pass and Fail ending variants take the same amount of time to complete.
+    - Example: If the Fail storyboard ends at time 20000 and the Pass storyboard ends at time 25000, the game will wait until time 25000 even if the player is in the Fail State (all objects will disappear). Therefore, it is best to ensure that both Pass and Fail ending variants take the same amount of time to complete.
   - Events will continue even if the user skips to the results screen early, and the audio produced by the storyboard can still be heard.
-- When in the Beatmap Editor's Design tab, the current time in milliseconds is displayed. Press Ctrl+C to copy the current time to your clipboard.
+- When in the beatmap editor's design tab, the current time in milliseconds is displayed. Press `Ctrl` + `C` to copy the current time to the clipboard.
```
Comment (The-Last-Cookie) [inline wiki/Storyboard/Scripting/General_Rules/en.md:102]:
```suggestion
- When in the beatmap editor's `Design` tab, the current time in milliseconds is displayed. Press `Ctrl` + `C` to copy the current time to the clipboard.
```

**Review by TicClick — COMMENTED** [review]
sorry that it took too long for me to post. while the update is alright-ish (it certainly makes the article better and a bit more smooth to read), it's not a cleanup -- many issues are still present, sentences need to be connected in a better way, it's still written in the guide form almost solely, etc, etc.

overall, it will be of little use if/when https://github.com/ppy/osu-wiki/issues/4827 happens, which brings (at least that is my expectation) much more structural changes to the table -- perhaps this article itself will vanish as a result. I'm honestly not a fan of discarding one's work but in my opinion it unfortunately would take much more to get the article into shape -- likely a rewrite from the ground-up, which is unlikely to be prioritized at the moment

**Comment by Walavouchey** [comment]
it is clean-up, albeit a partial one. i won't be reviewing this, but this can be merged as-is or with minimal review

> Check if the mention of `Taiko` and `Catch the Beat` is intentional (--> `osu!taiko` and `osu!catch` instead)

these should indeed be "osu!taiko" and "osu!catch"

"Design" (tab) shouldn't be in a block quote, see https://osu.ppy.sh/wiki/en/Client/Beatmap_editor/Compose and https://osu.ppy.sh/wiki/en/Client/Beatmap_editor/Timing

File: wiki/Storyboard/Scripting/General_Rules/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Storyboard/Scripting/General_Rules/en.md:25]: probably not, but it's a task for another time

File: wiki/Storyboard/Scripting/General_Rules/en.md (hunk 3 above)
Comment (TicClick) [reply wiki/Storyboard/Scripting/General_Rules/en.md:29]: let's keep it emphasized for now

**Comment by TicClick** [comment]
> Keyboard shortcuts are placed into two `. I'm not sure if this applies to image descriptions as well (see SBS_Time.jpg).

no -- they don't support Markdown

**Review by TicClick — APPROVED** [review]

**Pull request merged**